asllib/README.mld

```diff
@@ -1,18 +1,20 @@
 {0 Getting started with ASLRef}
 
 {1 Disclaimer}
-This material covers both ASLv0 (viz, the existing ASL pseudocode language
-which appears in the Arm Architecture Reference Manual) and ASLv1, a new,
-experimental, and as yet unreleased version of ASL.
+This material covers:
 
-This material is work in progress, more precisely at pre-Alpha quality as
+- ASLv0, viz the existing ASL pseudocode language which appears in the Arm Architecture Reference Manual:
+https://developer.arm.com/documentation/ddi0487/latest
```

Comment:
nit: perhaps we could use `odoc`'s link syntax, which according to [this cheatsheet](https://ocaml.github.io/odoc/odoc/cheatsheet.html) is:
```
{:https://www.example.com}
```
(similarly below)